src/arguments.js

```diff
@@ -89,18 +92,29 @@ const isValueBoolean = (string) => {
 };
 
 /**
- * Checks if argument string has any types by
- * additional chars after property name
+ * Get type of an argument (string, number, null (for flags), etc.)
  * @param {string} string - argument string without description
- * @return {bool} - result of check
+ * @param {object} context - internal parser config
+ * @return {string} argumentType - type of argument
  */
-const isPropertyTyped = (string) => {
-    let isType = false;
+const getPropertyType = (string, context) => {
+    let isFlag = true;
+    let argumentType = 'default';
```

Comment:
Possible do you mean a `null` value in argumentType ?

Reply:
@svoboda-rabstvo no, `null` have flags, and `default` will have all arguments if no types was found. `default` will be replaced later for value from `template/option.json`.

Reply:
I think it's no need. We can create implementation without it and it's complicated the implementation, also mislead users

Reply:
@svoboda-rabstvo done.